virtual_rainforest/models/animals/scaling_functions.py

```diff
@@ -0,0 +1,127 @@
+"""The `models.animals.scaling_functions` module contains a set of functions containing
+scaling equations" (relationships between body-mass and a train) required by the broader
+:mod:`~virtual_rainforest.models.animals` module
+"""  # noqa: D205, D415
+
+from math import ceil
+
+from virtual_rainforest.models.animals.constants import (
+    DAMUTHS_LAW_TERMS,
+    FAT_MASS_TERMS,
+    INTAKE_RATE_TERMS,
+    MEAT_ENERGY,
+    METABOLIC_RATE_TERMS,
+    MUSCLE_MASS_TERMS,
+)
+
+
+def damuths_law(mass: float) -> int:
+    """The function set initial population densities .
```

Comment:
I think the coefficients should go in the signature - something @dalonsoa encouraged. We have some slight differences in how this is done

* `soils` passes in a `coef` argument containing the whole constants dataclass as in [here](https://github.com/ImperialCollegeLondon/virtual_rainforest/blob/cec0f9c69f15f914ba886122d562f2495aac9a45/virtual_rainforest/models/soil/carbon.py#L157).

It would be something like:


* `abiotic` is more explicit in the low level functions about naming each constant as in [here](https://github.com/ImperialCollegeLondon/virtual_rainforest/blob/cec0f9c69f15f914ba886122d562f2495aac9a45/virtual_rainforest/models/abiotic/radiation.py#L199)

 ```suggestion
def damuths_law(
    mass: float,
    coef: float = DAMUTHS_LAW_TERMS.coefficient, 
    exp: float = DAMUTHS_LAW_TERMS.exponent
) -> int:
    """The function set initial population densities .
```